source/includes/get-started/quickstart.rb

```diff
@@ -0,0 +1,31 @@
+# start-bundler
+require 'bundler/inline'
+
+gemfile do
+  source 'https://rubygems.org'
+  gem 'mongo'
+end
+# end-bundler
+
+# start-query
+uri = '<connection string>'
+
+begin
+  client = Mongo::Client.new(uri)
+
+  database = client.use('sample_mflix')
+  movies = database[:movies]
+
+  # Queries for a movie that has the title 'Back to the Future'
+  query = { title: 'Back to the Future' }
+  movie = movies.find(query).first
+
+  # Prints the movie document
+  puts movie
+
+  client.close
+
+rescue => e
+  raise "Unable to find the document due to the following error: #{e}"
+end
```

Comment:
```suggestion
ensure
  client&.close
end
```

I do not think we need to catch errors here only to print them - uncatched errors will be printed to stderr anyways. Instead, we may put closing the client into `ensure`, this is a good practice. We also use `&` in the `client&.close` because `client` can be `nil` theoretically (if client creation failed).